superset/db_engine_specs.py

```diff
@@ -737,12 +761,20 @@ def handle_cursor(cls, cursor, query, session):
         while polled:
             # Update the object and wait for the kill signal.
             stats = polled.get('stats', {})
-
             query = session.query(type(query)).filter_by(id=query.id).one()
             if query.status in [QueryStatus.STOPPED, QueryStatus.TIMED_OUT]:
                 cursor.cancel()
                 break
 
+            if (
+                config.get('PREFETCH_ASYNC') and
+                (not query.has_loaded_early)
+            ):
+                query.has_loaded_early = True
+                limit = config.get('PREFETCH_ROWS')
```

Comment:
`prefetch_count` would be a better name than `limit` as limit has different meaning around limiting the query itself.